src/components/scene/reflection.js

```diff
@@ -61,9 +60,9 @@ module.exports.Component = register('reflection', {
     this.xrLightProbe = null;
     if (this.probeLight) {
       this.probeLight.components.light.light.intensity = 0;
+      this.needsVREnvironmentUpdate = true;
```

Comment:
we still need to move these?

Reply:
I am not sure: on one hand I do not see why the VR environment needs to be updated again if no light probes have been added, on the other hand recalculating should not have any effect if the light conditions have not changed, e.g. these line should be harmless...

However (full disclosure), in a use case of mine this does make a difference, specifically this happens: the scene starts "bright enough", but if I toggle the fullscreen mode, upon exiting fullscreen this becomes suddenly much darker.

I am unfortunately unable to reproduce this behavior on a public domain asset at the moment. I will remove the moving of these lines for the time being and will open a different PR if I manage to collect further understanding of the issue. So far I believe it has to do to light conditions at start of the scene being different from those at a later time (e.g. after a model has loaded).